packages/common/crypto/src/pkcs1/utils.ts

```diff
@@ -38,25 +38,22 @@ export function rsaep(n: BigNumber, e: BigNumber, m: BigNumber): BigNumber {
   return powerMod(m, e, n);
 }
 
-function powerMod(n: BigNumber, p: BigNumber, m: BigNumber): BigNumber {
+export function powerMod(n: BigNumber, p: BigNumber, m: BigNumber): BigNumber {
   // p === 1
   if (p.eq(new BigNumber(1))) {
     return n;
   }
 
+  // const t = power_mod(n, p >> 1n, m);
+  const t = powerMod(n, p.shrn(1), m);
+  const squared = t.mul(t).mod(m); // (t * t) % m;
+
   // p % 2 === 0
   if (p.mod(new BigNumber(2)).eq(new BigNumber(0))) {
-    // const t = power_mod(n, p >> 1n, m);
-    const t = powerMod(n, p.shrn(1), m);
-
-    return t.mul(t).mod(m); // (t * t) % m;
+    return squared;
   }
 
-  // const t = power_mod(n, p >> 1n, m);
-  const t = powerMod(n, p.shrn(1), m);
-
-  // return (t * t * n) % m;
-  return t.mul(t).mul(n).mod(m);
+  return squared.mul(n).mod(m); // (t * t * n) % m;
 }
 
```

Comment:
There is an comment at the top of this file as below. 
If We keep the original code and sync, I think **it will be less confusing when maintaining it in the future**, so I don't think We should make any changes, what do you think?

>// 소스코드의 주석은 특별한 언급이 없다면 god_crypto의 원본 소스코드 입니다
// reference: https://github.com/invisal/god_crypto/blob/25ec83d77c73a5eec1f763fbc660a9442ce0b65e/src/rsa/rsa_internal.t#L73

Reply:
I agree with you.